### ISSUE TITLE
Custom Tabular Exporter > Upload > A new Google Fusion table 500 ERROR

### ISSUE THREAD

**Issue by baditaflorin** [OpenRefine/OpenRefine#979]
HTTP ERROR 500

Problem accessing /extension/gdata/authorize. Reason:

 com.google.common.primitives.UnsignedInteger.asUnsigned(I)Lcom/google/common/primitives/UnsignedInteger;

Caused by:

java.lang.NoSuchMethodError: com.google.common.primitives.UnsignedInteger.asUnsigned(I)Lcom/google/common/primitives/UnsignedInteger;
	at com.google.api.client.util.Data.<clinit>(Data.java:81)
	at com.google.api.client.util.FieldInfo.<init>(FieldInfo.java:131)
	at com.google.api.client.util.FieldInfo.of(FieldInfo.java:104)
	at com.google.api.client.util.ClassInfo.<init>(ClassInfo.java:174)
	at com.google.api.client.util.ClassInfo.of(ClassInfo.java:92)
	at com.google.api.client.util.GenericData.<init>(GenericData.java:79)
	at com.google.api.client.util.GenericData.<init>(GenericData.java:61)
	at com.google.api.client.http.GenericUrl.<init>(GenericUrl.java:97)
	at com.google.api.client.auth.oauth2.AuthorizationRequestUrl.<init>(AuthorizationRequestUrl.java:99)
	at com.google.api.client.auth.oauth2.AuthorizationCodeRequestUrl.<init>(AuthorizationCodeRequestUrl.java:60)
	at com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeRequestUrl.<init>(GoogleAuthorizationCodeRequestUrl.java:96)
	at com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeRequestUrl.<init>(GoogleAuthorizationCodeRequestUrl.java:82)
	at com.google.refine.extension.gdata.GDataExtension.getAuthorizationUrl(GDataExtension.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:161)
	at org.mozilla.javascript.NativeJavaMethod.call(NativeJavaMethod.java:247)
	at org.mozilla.javascript.optimizer.OptRuntime.call2(OptRuntime.java:76)
	at org.mozilla.javascript.gen.c10._c2(file:/F:/github/OpenRefine-modificat/main/webapp/../../extensions/gdata/module/MOD-INF/controller.js:105)
	at org.mozilla.javascript.gen.c10.call(file:/F:/github/OpenRefine-modificat/main/webapp/../../extensions/gdata/module/MOD-INF/controller.js)
	at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:398)
	at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3065)
	at org.mozilla.javascript.gen.c10.call(file:/F:/github/OpenRefine-modificat/main/webapp/../../extensions/gdata/module/MOD-INF/controller.js)
	at edu.mit.simile.butterfly.ButterflyModuleImpl$Controller.process(ButterflyModuleImpl.java:399)
	at edu.mit.simile.butterfly.ButterflyModuleImpl$Controller.run(ButterflyModuleImpl.java:377)
	at org.mozilla.javascript.Context.call(Context.java:515)
	at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:507)
	at edu.mit.simile.butterfly.ButterflyModuleImpl.processScript(ButterflyModuleImpl.java:650)
	at edu.mit.simile.butterfly.ButterflyModuleImpl.process(ButterflyModuleImpl.java:427)
	at edu.mit.simile.butterfly.Butterfly.service(Butterfly.java:516)
	at com.google.refine.RefineServlet.service(RefineServlet.java:200)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1166)
	at org.mortbay.servlet.UserAgentFilter.doFilter(UserAgentFilter.java:81)
	at org.mortbay.servlet.GzipFilter.doFilter(GzipFilter.java:132)
	at org.mortbay.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1157)
	at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:388)
	at org.mortbay.jetty.security.SecurityHandler.handle(SecurityHandler.java:216)
	at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:182)
	at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:765)
	at org.mortbay.jetty.webapp.WebAppContext.handle(WebAppContext.java:418)
	at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
	at org.mortbay.jetty.Server.handle(Server.java:326)
	at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:542)
	at org.mortbay.jetty.HttpConnection$RequestHandler.headerComplete(HttpConnection.java:923)
	at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:547)
	at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:212)
	at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
	at org.mortbay.jetty.bio.SocketConnector$Connection.run(SocketConnector.java:228)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

Powered by Jetty://

**Comment by tfmorris** [comment]
I suspect this is a version mismatch on a dependency such as Guava.

**Comment by tfmorris** [comment]
This happens with v2.6-rc1 and needs to be fixed before release.

**Comment by jackyq2015** [comment]
Agreed! Downgrading the  guava to 1.3 solves the problem.

**Comment by tfmorris** [comment]
This method was deprecated, then removed in Guava 15.  We had updated to Guava 18, from Guava 13, but I've reverted that with bfc39d9e8c22edc035929f24d6b34adf2d8c9335.

We are probably going to need to update to more recent versions google-api-client and related libs to get support for replacements for deprecated APIs (e.g. Google Spreadsheets creation API) We're using 1.13.1-beta and the current version is 1.19.1.  

Unfortunately, there are a fairly intricate set of dependences for the Freebase and GData extensions which may be in conflict with each other, but we'll leave that for another day.
